recipes/corona_lineage_dynamics/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+${CXX} --std=c++17 scripts/count_lineages_country.cpp -o scripts/count_lineages_country -lstdc++ -lboost_regex ${CXXFLAGS} ${LDFLAGS}
+
+Rscript -e 'install.packages(c("lifecycle", "scales", "withr", "knitr"), repos="https://cloud.r-project.org");'
+Rscript -e 'devtools::install_github("rstudio/d3heatmap");'
```

Comment:
`master` branch is a moving target. Every build of this recipe may use different code of this dependency!
Better depend on:
- https://anaconda.org/conda-forge/r-d3heatmap

Reply:
Actually that was a very good suggestion! Moved into dependencies.